src/utils/CoreServicesUtils.ts

```diff
@@ -0,0 +1,37 @@
+export const CoreServicesGeoNamesMapping: any = { // eslint-disable-line @typescript-eslint/no-explicit-any

```

Comment:
who maintains this? , if the list changes how we will know?, how do we know this is the right version and mapping?